spec/output/generator_spec_elixir/nesting.x/my_union.ex

```diff
@@ -0,0 +1,74 @@
+defmodule MyXDR.MyUnion do
+  @moduledoc """
+  Automatically generated by xdrgen
+  DO NOT EDIT or your changes may be overwritten
+
+  Target implementation: elixir_xdr at https://hex.pm/packages/elixir_xdr
+
+  Representation of Stellar `MyUnion` type.
+  """
+
+  @behaviour XDR.Declaration
+
+  alias MyXDR.{UnionKey,
+    MyUnionOne,
+    MyUnionTwo,
+    XDR.Type.Void
+  }
```

Comment:
1. Add a break line for the first module of the alias.
2. Currently, StellarBase has its own `Void` type. Could we generate it when in the `.x` file is used?

If so, the output of this chunk of code should be:
```elixir
  alias MyXDR.{
    UnionKey,
    MyUnionOne,
    MyUnionTwo,
    Void
  }
```